.github/workflows/issue-transferred.yml

```
@@ -10,6 +10,8 @@ jobs:
   issue-transferred:
     name: Issue Transferred
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
```

```suggestion
    permissions: {}
```

I believe this doesn't need any perms since it's using a GH app token, and it was already inheriting no perms.